src/Kunstmaan/MediaBundle/Controller/MediaController.php

```diff
@@ -270,12 +263,19 @@ public function bulkUploadSubmitAction(Request $request, $folderId)
         }
 
 
-        // Return Success JSON-RPC response
-        return new JsonResponse(array(
+        // Send headers making sure that the file is not cached (as it happens for example on iOS devices)
+        $response = new JsonResponse([
             'jsonrpc' => '2.0',
             'result'  => '',
             'id'      => 'id'
-        ));
+        ], JsonResponse::HTTP_OK, [
+            'Expires' => 'Mon, 26 Jul 1997 05:00:00 GMT',
+            'Last-Modified' => '' . gmdate('D, d M Y H:i:s') . ' GMT',
```

Comment:
```'Last-Modified' => '' . gmdate('D, d M Y H:i:s') . ' GMT'```
to
``` 'Last-Modified' => gmdate('D, d M Y H:i:s') . ' GMT'```

Reply:
i did a find and replace colon space with ' => ',  to split the header strings into two :-P hold on!